quint/src/cliCommands.ts

```diff
@@ -517,6 +517,33 @@ export async function compile(typechecked: TypecheckedStage): Promise<CLIProcedu
     return cliErr(`module ${mainName} does not exist`, { ...typechecked, errors: [], sourceCode: new Map() })
   }
 
+  const hasInit = typechecked.resolver.collector.getDefinition(args.init) !== undefined
+  const hasStep = typechecked.resolver.collector.getDefinition(args.step) !== undefined
+  const flattenRequested = args.flatten == true
+
+  const targetIsTla = args.target === 'tlaplus'
+
+  // check which core definitions are missing
+  let flattenRequires: string | null = null
```

Comment:
Can we call this `missingFlattenDefs`? I find the `requires` confusing in this context